docgen/parser/model.go

```diff
@@ -53,6 +53,33 @@ func (f *File) HasServices() bool {
 	return len(f.Services) > 0
 }
 
+func (f *File) CheckSwagger() error {
```

Comment:
these are necessary so that the markdown result builds, and does not have empty headers (which breaks FE)

Reply:
please document this in the godoc.